setup.cfg

```diff
@@ -1,6 +1,7 @@
 [flake8]
 exclude = .git,.eggs,build,venv,env
-max-line-length = 79
+max-line-length = 88
```

Comment:
this parameter is changed because black recommends using 88 as line length, ref - https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#labels-line-length

Reply:
Maybe, but we have enforced 79 in all of our code bases, so maybe it's better to stick with that